pkg/backup/util/aws_ebs.go

```diff
@@ -204,6 +205,68 @@ func (e *EC2Session) DeleteSnapshots(snapIDMap map[string]string, deleteRatio fl
 	return nil
 }
 
+func (e *EC2Session) DeleteVolumes(volumeIDs []string) error {
```

Comment:
We should only delete volumes, which are unattached. How is the validated here.

Reply:
If the volume is attached, the delete request will fail. It means, we can't delete an attached volume. In my test, if the tikv pod is running, the volume can't be deleted. This is the reason why I deleted the tikv statefulset in my test steps.

Reply:
Got it. Basically we rely in aws api contract to prevent attached volumes from getting deleted. `Deletes the specified EBS volume. The volume must be in the available state (not attached to an instance).`

https://docs.aws.amazon.com/cli/latest/reference/ec2/delete-volume.html